### PR TITLE
Add documentation for ActiveMQ Scaler

### PR DESCRIPTION
**Pull request by melisatanrverdi**
Signed-off-by: melisatanrverdi <melisatanrverdi@gmail.com>

Provide documentation for ActiveMQ scaler.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to kedacore/keda#2305

### CODE REVIEW

File: content/docs/2.6/scalers/activemq.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+### Trigger Specification
+
+This specification describes the `activemq` trigger that scales based on a ActiveMQ Queue.
+
+```yaml
+triggers:
+  - type: activemq
+    metadata:
+      managementEndpoint: "activemq.activemq-test:8161"
+      destinationName: "testQueue"
```
Comment (tomkerkhove) [inline content/docs/2.6/scalers/activemq.md:19]: Would it make sense to just call this `queueName`?

**Review by tomkerkhove — COMMENTED** [review]
Added a quick remark. Would you mind aligning with our [doc guidance](https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler) please?

File: content/docs/2.6/scalers/activemq.md (hunk 1 above)
Comment (melisatanrverdi) [reply content/docs/2.6/scalers/activemq.md:19]: It defined as destination name in Rest API template. I also thought to change but I wasn't sure that it'll make confusion or not so I kept as destination name. Do you suggest change it as queueName?

Comment (melisatanrverdi) [reply content/docs/2.6/scalers/activemq.md:19]: It being mentioned as destination in ActiveMQ documents. For instance: https://activemq.apache.org/how-do-i-create-new-destinations

Comment (melisatanrverdi) [reply content/docs/2.6/scalers/activemq.md:19]: This is the Rest API template that I monitor message count in scaler. It use as destinationName in the template: "http://<<managementEndpoint>>/api/jolokia/read/org.apache.activemq:type=Broker,brokerName=<<brokerName>>,destinationType=Queue,destinationName=<<destinationName>>/QueueSize". 

This document explains object names  more clear under the ActiveMQ MBeans Reference title: https://activemq.apache.org/jmx

Comment (zroubalik) [reply content/docs/2.6/scalers/activemq.md:19]: Great, I'd keep this aligned with ActiveMQ

File: content/docs/2.6/scalers/activemq.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- ActiveMQ Scaler polls the ActiveMQ Rest API to monitor message count of target queue. Currently, the scaler supports basic authentication. `username` and `password` are **required**. See [Authentication Parameters](#authentication-parameters) below. 
+
+### Authentication Parameters
+
+You can use TriggerAuthentication CRD to configure the username and password to connect to the management endpoint.
+
+**Username and Password based authentication:**
+
+- `username` - Required. The username to use to connect to the management endpoint of ActiveMQ.
+- `password` - Required. The password to use to connect to the management endpoint of ActiveMQ.
```
Comment (tomkerkhove) [inline content/docs/2.6/scalers/activemq.md:44]: Would you mind aligning with our [doc guidance](https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler) please?

**Review by tomkerkhove — APPROVED** [review]

File: content/docs/2.6/scalers/activemq.md (hunk 2 above)
Comment (tomkerkhove) [reply content/docs/2.6/scalers/activemq.md]: Can you align the auth parameters section as well please?

File: content/docs/2.6/scalers/activemq.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    targetQueueSize: "100"
+```
+
+**Parameter list:**
+
+- `managementEndpoint` - ActiveMQ management endpoint in format: `<hostname>:<port>`.
+- `destinationName` - Name of the queue to check for the message count.
+- `brokerName` - Name of the broker as defined in ActiveMQ.
+- `targetQueueSize` - Target value for queue length passed to the scaler. The scaler will cause the replicas to increase if the queue message count is greater than the target value per active replica. (Default: `10`, Optional)
+- `restAPITemplate` - Template to build REST API url to get queue size. (Default: `"http://<<managementEndpoint>>/api/jolokia/read/org.apache.activemq:type=Broker,brokerName=<<brokerName>>,destinationType=Queue,destinationName=<<destinationName>>/QueueSize"`, Optional)
```
Comment (arschles) [inline content/docs/2.6/scalers/activemq.md:30]: am I mistaken or is the template using the Go [text/template](https://godoc.org/text/template) syntax now?

Comment (melisatanrverdi) [reply content/docs/2.6/scalers/activemq.md]: Yes, I corrected default template syntax.

File: content/docs/2.6/scalers/activemq.md (hunk 2 above)
Comment (melisatanrverdi) [reply content/docs/2.6/scalers/activemq.md]: I aligned the auth parameters section.

**Review by tomkerkhove — APPROVED** [review]
LGTM, thanks a ton @melisatanrverdi!🎉

**Review by zroubalik — APPROVED** [review]

**Pull request merged**